### ISSUE TITLE
Question: Integration with Conda/Jupyter?

### ISSUE THREAD

**Issue by BradKML** [pyston/pyston_v2#21]
Since Pyston is fast, would it not be good for Conda and Jupyter to know this project and pick this up?

**Comment by kmod** [comment]
Yes we would love to! I'm not familiar with Conda but I was told that we don't need their involvement if we want to create a Pyston option via Conda, and it'd be great to have that.

**Comment by BradKML** [comment]
https://jupyter.org/ https://docs.conda.io/en/latest/

**Comment by starlinq** [comment]
FYI: [https://github.com/astrorama/pyston-feedstock](https://github.com/astrorama/pyston-feedstock)

**Comment by BradKML** [comment]
@starlinq will keep an eye on this, but other than that, are there any other implementations of Conda porting?

**Comment by undingen** [comment]
Conda integration is now very likely which will make the user experience much nicer: 
https://blog.pyston.org/2021/08/30/pyston-team-joins-anaconda/
But we don't have a timeline yet.

**Comment by starlinq** [comment]
> @starlinq will keep an eye on this, but other than that, are there any other implementations of Conda porting?

@BrandonKMLee I could not find anything else like that

**Comment by marscher** [comment]
Eventually you should consider publishing within the conda-forge channel to provide Pyston as an interpreter. Building C++ code with conda-build is working like a charm, as it provides lots of libraries (llvm) and compilers (for all major platforms).
I guess this woud not only makes the user experience much better, but would also attract more developers to try to support Pyston/try it out and contribute to this project. That being said, I hope that this issue would climb up in the priority list a bit.

The already linked conda recipe for building Pyston by astrorama should be a good starting point.

**Comment by kmod** [comment]
fwiw the linked feedstock is for an unrelated project also called Pyston

But yes we are actively working on this! Check out https://blog.pyston.org/2021/10/26/pyston-roadmap/ for more details
